### ISSUE TITLE
Support @SuppressWarnings on local variables

### ISSUE THREAD

**Issue by mkobit** [CodeNarc/CodeNarc#197]
I have a use case where I need to suppress a CodeNarc rule in a script. I would like to be able to do something like:

```groovy
@SuppressWarnings('UnusedVariable')
final myVar = thing.findVar()
```

I am using `0.26.0` from Gradle and would like to use my normal ruleset in general, with possible some disabled in scripts, but make it visible in those scripts rather than with with `codenarc.groovy` configuration I am using.

Relevant discussion from before: https://sourceforge.net/p/codenarc/mailman/message/28420205/

**Comment by chrismair** [comment]
For what it's worth, the current design of the **@SuppressWarnings** handling does not include variables:

> You can use the @SuppressWarnings annotation on a class, method, constructor, field or property to suppress one or more CodeNarc rules within its source code.

I am not saying it is impossible to enhance that, but just that it was an intentional limitation.

**Comment by mkobit** [comment]
Good point! I didn't see that originally, but thanks for highlighting it.

For reference of tohers that may come to this issue, I found the documentation you quoted at http://codenarc.sourceforge.net/codenarc-configuring-rules.html

**Comment by mkobit** [comment]
Updated the title to better reflect the request.

**Comment by aadamovich** [comment]
Submitted a pull request that should make @SuppressWarnings work on local variables

**Comment by chrismair** [comment]
Fixed by #206
